lib/components/user-input/Input.tsx

```diff
@@ -26,6 +26,7 @@ export type InputProps = {
   onChangeEvent?: (event: ChangeEvent<HTMLInputElement>) => void,
   className?: string,
   onEditCompleted?: (text: string) => void,
+  onEditCompletedEvent?: (event: ChangeEvent<HTMLInputElement>) => void,
```

Comment:
```suggestion
  onEditCompleted?: (text: string, event: ChangeEvent<HTMLInputElement>) => void,
```
Same goes for `onChangeEvent` a few lines above. I guess we could quickly refactor this as well.